### ISSUE TITLE
Add list of contributing labs and individuals

### ISSUE THREAD

**Issue by kachergis** [manybabies/manybabies.github.io#10]
MB1 authors + other significant individual contributors -- where should it go? Under a "Collaborators" heading on the About page?

**Comment by kachergis** [comment]
MB1 authors: Christina Bergmann, Michael C. Frank, Nayeli Gonzalez, Elika Bergelson, Alejandrina Cristia, Brock Ferguson, Melissa Kline, Melanie Soderstrom, Daniel Yurovsky, Krista Byers-Heinlein, Robin Panneton, Caroline Floccia, Casey Lew-Williams, Kiley Hamlin, Mohinish Shukla
- what other individuals should be added?

**Comment by kristabh** [comment]
A full list of contributors to MB1 is here
https://docs.google.com/spreadsheets/d/1O225ib9wFjUjZJfz9jqwRkfFYaCGXrtDYnmkK9NM3r8/edit?usp=sharing

**Comment by kristabh** [comment]
And here are the MB1B contributors (large overlap with MB1, but maybe not perfect):

https://docs.google.com/spreadsheets/d/13_1EEUCYhX5NY5PynSchVFVlAb3BT5KQQjK2XxJft8I/edit?usp=sharing

**Comment by christinabergmann** [comment]
Re: Where. I'd say if we have project subpages (i.e. one for MB1, 2, etc), maybe it would be possible to have a nicely formatted version of the spreadsheet? It's a big blob of text with little info otherwise and I do like the who-did-what extra layer of information in the spreadsheet.

**Comment by RodrigoDalBen** [comment]
I believe that this was covered on the updated website + shiny dashboard/map.

**Comment by christinabergmann** [comment]
What it doesn't cover is type of contribution (e.g. data, analysis), but that's fine for now. Might be interesting to consider further in the future, just to flag this as possibility. (e.g. color coding? different symbols)

**Comment by RodrigoDalBen** [comment]
Right, we have this field in the form, but getting this info will be time consuming (a lot of e-mails + reminders). I think the future general registration will help us deal with that.
